model/task_lifecycle.go

```diff
@@ -1203,8 +1282,18 @@ func evalLinearStepback(ctx context.Context, t *task.Task, caller, status string
 
 // evalBisectStepback performs bisect stepback on the task.
 func evalBisectStepback(ctx context.Context, t *task.Task, caller, status string, deactivatePrevious bool) error {
```

Comment:
We also will need to pass in `stepback` as a bool param to this function just as [evalLinearStepback does](https://github.com/evergreen-ci/evergreen/pull/7124/files#diff-4bef865c41d6d16e1056e61806c23fb8bbc896b3739472dd60f0dd463ef5d2b9R1243) to ensure that we're [respecting](https://github.com/evergreen-ci/evergreen/pull/7124/files#diff-4bef865c41d6d16e1056e61806c23fb8bbc896b3739472dd60f0dd463ef5d2b9R1251-R1253) if stepback is set to false for the project/BV/task in question.